libs/ballot-interpreter/src/hmpb-rust/interpret.rs

```diff
@@ -327,12 +314,12 @@ pub fn interpret_ballot_card(
     };
 
     let (side_a_grid_result, side_b_grid_result) = par_map_pair(
-        (&side_a.image, &mut side_a_debug),
-        (&side_b.image, &mut side_b_debug),
-        |(image, debug)| {
```

Comment:
Changes along these lines are just to support passing a `BallotImage` instead of a `GrayImage` in order to get the threshold alongside the image.